aruco_poses:
  front_mid:
    id: 0
    translation: [0.0, 0.0, 0.0]
    rotation: [0.0, 0.0, 0.0, 1.0]
  front_right:
    id: 1
    translation: [0.0, 0.0, 0.0]
    rotation: [0.0, 0.0, 0.0, 1.0]
  front_left:
    id: 2
    translation: [0.0, 0.0, 0.0]
    rotation: [0.0, 0.0, 0.0, 1.0]
  left:
    id: 4
    translation: [0.0, 0.0, 0.0]
    rotation: [0.0, 0.0, 0.0, 1.0]
  right:
    id: 5
    translation: [0.0, 0.0, 0.0]
    rotation: [0.0, 0.0, 0.0, 1.0]
  back_mid:
    id: 3
    translation: [0.0, 0.0, 0.0]
    rotation: [0.0, 0.0, 0.0, 1.0]
  back_right:
    id: 6
    translation: [0.0, 0.0, 0.0]
    rotation: [0.0, 0.0, 0.0, 1.0]
  back_left:
    id: 7
    translation: [0.0, 0.0, 0.0]
    rotation: [0.0, 0.0, 0.0, 1.0]